packages/app/src/app/pages/Dashboard/index.tsx

```diff
@@ -34,6 +35,7 @@ export const Dashboard: FunctionComponent = () => {
   const { browser, notificationToast } = useEffects();
   const actions = useActions();
   const { subscription } = useWorkspaceSubscription();
+  const [, trackVisit] = useDashboardVisit();
```

Comment:
Is it better to return an object from the `useDashboardVisit` hook instead so we don't need the `[, ...]`? Not sure if that has a name 😄.

Reply:
Definitely! Will change